src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacilityStatic/ClosestFacilityStatic.cs

```diff
@@ -0,0 +1,271 @@
+using Esri.ArcGISRuntime.Tasks.NetworkAnalysis;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+
+namespace ArcGISRuntime.Samples.ClosestFacilityStatic
+{
+    [Activity(Label = "ClosestFacilityStatic")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Closest facility (static)",
+        "Network Analysis",
+        "Demonstrates how to solve a Closest Facility Task to find the closest route between facilities and incidents.",
+        "Click the solve button to find the closest facility to every incident.")]
+    public class ClosestFacilityStatic : Activity
+    {
+        // Create a MapView.
+        private MapView _myMapView = new MapView();
+
+        // Add buttons for the UI.
+        private Button _solveRoutesButton;
+        private Button _resetButton;
+
+        // Used to display route between incident and facility to mapview.
+        private List<SimpleLineSymbol> _routeSymbols;
+
+        // Solves task to find closest route between an incident and a facility.
+        private ClosestFacilityTask _task;
+
+        // Table of all facilities.
+        private ServiceFeatureTable _facilityTable;
+
+        // Table of all incidents.
+        private ServiceFeatureTable _incidentTable;
+
+        // Feature layer for facilities.
+        private FeatureLayer _facilityLayer;
+
+        // Feature layer for incidents.
+        private FeatureLayer _incidentLayer;
+
+        // Uri for facilities feature service.
+        private Uri _facilityUri = new Uri("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0");
+
+        // Uri for incident feature service.
+        private Uri _incidentUri = new Uri("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Incidents/FeatureServer/0");
+
+        // Uri for the closest facility service.
+        private Uri _closestFacilityUri = new Uri("http://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility");
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Closest facility (static)";
+
+            // Create the UI
+            CreateLayout();
+
+            // Initialize the app
+            Initialize();
+        }
+
+        private void CreateLayout()
+        {
+            // Create a new layout for the entire page
+            LinearLayout layout = new LinearLayout(this)
+            {
+                Orientation = Orientation.Vertical
+            };
+
+            // Create a new layout for the toolbar (buttons)
+            LinearLayout toolbar = new LinearLayout(this)
+            {
+                Orientation = Orientation.Horizontal
+            };
+
+            // Create a button to solve the route and add it to the toolbar
```

Comment:
Don't forget to be consistent about periods at the end of sentences in comments.